src/applications/vaos/appointment-list/components/AppointmentsPage/ScheduleNewAppointmentRadioButtons.jsx

```diff
@@ -0,0 +1,74 @@
+import React, { useState } from 'react';
+import RadioButtons from '@department-of-veterans-affairs/component-library/RadioButtons';
+import { Link } from 'react-router-dom';
+import recordEvent from 'platform/monitoring/record-event';
+import { GA_PREFIX } from 'applications/vaos/utils/constants';
+
+export default function ScheduleNewAppointmentRadioButtons({
+  showCheetahScheduleButton,
+  startNewAppointmentFlow,
+}) {
+  const [radioSelection, setRadioSelection] = useState();
+  const [radioSelectionEvent, setRadioSelectionEvent] = useState();
+
+  function radioOptions() {
+    const optionsArray = [
+      {
+        value: 'new-appointment',
+        label: 'Primary or specialty care',
+      },
+    ];
+
+    if (showCheetahScheduleButton) {
+      optionsArray.push({
+        value: 'new-covid-19-vaccine-booking',
+        label: 'COVID-19 vaccine',
+      });
+    }
+    return optionsArray;
+  }
+
+  return (
+    <>
+      <h2 className="vads-u-padding-bottom--0">Schedule a new appointment</h2>
```

Comment:
This h2 looks too large to me on the current homepage:

![Screen Shot 2021-03-11 at 11 24 51 AM](https://user-images.githubusercontent.com/634932/110819715-7109c580-825c-11eb-8145-9254f92b8be4.png)

Reply:
Good catch, added the h3 class to match the other h2 designs on that page.